docs/start/getting-started/fragments/angular/auth.md

```diff
@@ -45,7 +45,7 @@ Add the `amplify-authenticator` component to the top of `src/app/app.component.h
 Run the app to see the new Authentication flow protecting the app:
 
 ```bash
-ng serve
+npm start
```

Comment:
Nit: These things are *basically* the same now that Angular CLI uses webpack but they used to be different. For older Angular CLIs this won't be the same result.